src/popover/popover.directive.ts

```diff
@@ -153,8 +153,12 @@ export class PopoverContainer implements AfterViewInit, OnChanges, OnDestroy {
 					const computedStyle = getComputedStyle(this.caretRef);
 					const offset = computedStyle.getPropertyValue("--cds-popover-offset");
 					const height = computedStyle.getPropertyValue("--cds-popover-caret-height");
-					this.caretOffset = (offset?.includes("px") ? Number(offset.split("px", 1)[0]) : Number(offset.split("rem", 1)[0]) * 16) || 10;
-					this.caretHeight = (height?.includes("px") ? Number(height.split("px", 1)[0]) : Number(height.split("rem", 1)[0]) * 16) || 6;
+					this.caretOffset = (offset?.includes("px")
+						? Number(offset.split("px", 1)[0])
+						: Number(offset.split("rem", 1)[0]) * 16) || 10;
+					this.caretHeight = (height?.includes("px")
+						? Number(height.split("px", 1)[0])
+						: Number(height.split("rem", 1)[0]) * 16) || 6;
```

Comment:
Seems harder to read now? Could just be how it's coded that makes it harder to read?

Thoughts on this?

Reply:
<img width="736" alt="Screenshot 2025-02-26 at 14 33 13" src="https://github.com/user-attachments/assets/76db0c7e-54c6-418a-8c3e-cce062fbc389" />

This is because of the line length too. Another case that was not caught in tslint. 
I am used to seeing this structure because of Prettier in my other projects, so for me this is actually more clear. 

We could slightly increase the line length to resolve a couple of changes I did?